build.gradle

```diff
@@ -108,6 +121,12 @@ rat {
   // resource files for test
   excludes.add("**/test**/resources/**")
 
+  // resources
```

Comment:
we should exclude `**/dependencies/**`

Reply:
That should be covered by the line in `.gitignore` right? I didn't want to duplicate things that were already handled by the git ignore file, but I can if you think it's necessary.

Reply:
from what I've seen, it's much faster when you explicitly add it here. So if you run `./gradlew rat` with and without the explicit exclude the build speed will change dramatically. Maybe it's not longer true after these changes

Reply:
That's really bizarre - the literally make the same call to `excludes.add` - I can't understand how it could be different.

Testing locally, there appears to be absolutely no difference between the two.